<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicRoom</class>
 <widget class="QMainWindow" name="MusicRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Room</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="TreeWidget_Category">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>231</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Musics</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Folder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PlayLists</string>
     </property>
     <item>
      <property name="text">
       <string>Happy</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Queue</string>
     </property>
    </item>
   </widget>
   <widget class="QListView" name="ListView_AudioTracks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::EditKeyPressed</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Audio Tracks:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_PlayPause">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Next">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Prev">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_Shuffle">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Suffle: Off</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Folder Selection</string>
    </property>
    <widget class="QPushButton" name="PushButton_Browse">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEdit_Path">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>D:/Music/Playlist 1</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="PushButton_Repeat">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RepeatMode: No-Repeat</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_MusicSlider">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #444;
    height: 6px;
    background: #222;
    margin: 0px;
    border-radius: 3px;
}

QSlider::handle:horizontal {
    background: #00bcd4;
    border: 1px solid #00acc1;
    width: 14px;
    height: 14px;
    margin: -5px 0; /* centers the handle */
    border-radius: 7px;
}

QSlider::sub-page:horizontal {
    background: #00acc1;
    border-radius: 3px;
}

QSlider::add-page:horizontal {
    background: #555;
    border-radius: 3px;
}
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Timer">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="ListView_PlayLists">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::EditKeyPressed</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PlayLists:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_AddTrackToPlayList">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>==&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_SwitchView">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1DB954;
      color: white;
      border: none;
      border-radius: 8px;
      padding: 10px 20px;
      font-size: 11px;
    }
    QPushButton:hover {
    background-color: #1ed760;
    }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="Widget_MusicDisplay">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="Label_AlbumCover">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="VisualizerWidget" name="VisualizerDisplay" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSlider" name="Slider_Volume">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>20</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical {
    background: #222;
    border: 1px solid #444;
    width: 6px;
    border-radius: 3px;
    margin: 0px;
}

QSlider::handle:vertical {
    background: yellow;
    border: 1px solid black;
    height: 14px;
    width: 14px;
    margin: 0 -5px; /* centers the handle horizontally */
    border-radius: 7px;
}

QSlider::sub-page:vertical {
    background: #555;
    border-radius: 3px;
}

QSlider::add-page:vertical {
    background: yellow;
	border: 0.8px solid black;
	border-radius: 3px;
}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_AddTrackToQueue">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add to Queue</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>211</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Create PlayList</string>
    </property>
    <widget class="QLineEdit" name="LineEdit_PlayListName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="PushButton_CreatePlayList">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #1DB954;
      color: white;
      border: none;
      border-radius: 8px;
      padding: 10px 20px;
      font-size: 13px;
    }
    QPushButton:hover {
    background-color: #1ed760;
    }</string>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBox_OnlineBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>431</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string>Online Box</string>
    </property>
    <widget class="QTextBrowser" name="TextBrowser_Messages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>411</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="TextEdit_Message">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="ListView_PlayLists_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>411</width>
       <height>141</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::EditKeyPressed</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Users:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Chat:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PushButton_Send">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>570</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::FocusPolicy::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VisualizerWidget</class>
   <extends>QWidget</extends>
   <header location="global">visualizerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
